uber/templates/schedule/schedule_base.html

```diff
@@ -0,0 +1,89 @@
+{% extends "base.html" %}
+
+{% block title %}Schedule{% endblock %}
+{% import 'schedule/schedule_macros.html' as schedule_macros with context %}
+
+{% block content %}
+    {% include "panels_common.html" %}
+    {% include "schedule_common.html" %}
+
+    <style type="text/css">
+        .schedule-table {
+          top: 132px;
+        }
+
+        body {
+            background-color: #ffffff;
+            background-image: none;
+        }
+    </style>
```

Comment:
this body-specific CSS is new and was required to remove the dating-sim background and fix the color on the public schedule page.  it's a little ugly here but gets the job done